esp32/ugfx_widgets.c

```diff
@@ -635,7 +635,13 @@ void BWgwinListDefaultDraw(GWidgetObject* gw, void* param) {
 				}
 			}
 		#endif
-		gdispGFillStringBox(gw->g.display, gw->g.x+x+LST_HORIZ_PAD, gw->g.y+y, iwidth-LST_HORIZ_PAD, iheight, qi2li->text, gw->g.font, text, fill, justifyLeft);
```

Comment:
It does?!

I just read the [API docs](http://api.ugfx.io/group___g_d_i_s_p.html#ga31df37ca1dcde14bd357ec08708b618f) and they say that text is always vertically centered and `justify` only adjusts horizontal. So either you're getting unintended correctness or the docs need to be updated :-).